### DISCUSSION TITLE
raspberry pi model b rev 2 (OLD MODEL)

### DISCUSSION THREAD

**Discussion by valsergi** [Koenkk/zigbee2mqtt#18269] [General]
Hello, I have a question, and I would like opinions...

I currently have a second zigbee2mqtt server running on a raspberry pi model b rev 2, with version 1.16.1 and it works great for me, it is my second server and it is installed in the storage room and it perfectly manages 5 sensors, 2 door, 1 bulb one for vibration and one for temperature and humidity, now I want to add a plug of those that measure consumption version and surely it will not be recognized, my question is the following... will this hardware support the update to the most recent server, if so How should I do it so as not to lose what I currently have configured... THANK YOU!!!

Sorry, I forgot to comment that I obviously have it integrated into HA with a different topic so that the two servers work separately and each one manages its sensors. The other server is at home on another more modern raspberry 3B+

**Comment by francisp2** [comment]
Linux install ? 

# Stop Zigbee2MQTT and go to directory
sudo systemctl stop zigbee2mqtt
cd /opt/zigbee2mqtt

# Backup configuration
cp -R data data-backup

# Update
git pull
npm ci

# Restore configuration
cp -R data-backup/* data
rm -rf data-backup

# Start Zigbee2MQTT
sudo systemctl start zigbee2mqtt
